packages/orbit-components/src/Toast/README.md

```diff
@@ -0,0 +1,62 @@
+# Toast
+
+To implement Toast component into your project you'll need to add the import:
+
+```jsx
+import ToastInit, { createToast } from "@kiwicom/orbit-components/lib/Toast";
+```
+
+It's better to use ToastInit at the root of your application once and you can use `createToast` from anywhere after
```

Comment:
Because it behaves almost in the same way as the normal provider. I do not think the example is needed, it's just copied from original `react-hot-toast` documentation, which is also recommended to add at the root.